class.photon.php

```diff
@@ -336,11 +336,11 @@ public static function filter_the_content( $content ) {
 					$width = $height = false;
 
 					// First, check the image tag
-					if ( preg_match( '#width=["|\']?([\d%]+)["|\']?#i', $images['img_tag'][ $index ], $width_string ) ) {
+					if ( preg_match( '#[\s|"|\']width=["|\']?([\d%]+)["|\']?#i', $images['img_tag'][ $index ], $width_string ) ) {
```

Comment:
If we're searching by the width property string vs the entire post_content, is it not clearer to do: 
```^width``` for the beginning of line?

<img width="395" alt="Screen Shot 2019-11-06 at 10 51 50 AM" src="https://user-images.githubusercontent.com/1270189/68328107-84fb5100-0083-11ea-8eec-2e008c709723.png">

Reply:
Unfortunately what we have here is the whole image tag, so the beginning of line is always `<img...` which wouldn't match with `^width`.

![image](https://user-images.githubusercontent.com/1233880/68375641-7ecea880-0147-11ea-8c49-f18abc4a3c08.png)

![image](https://user-images.githubusercontent.com/1233880/68375710-9efe6780-0147-11ea-9f9c-4a7f7f96b509.png)

Reply:
Can't we just match a literal space character?

```suggestion
					if ( preg_match( '# width=["|\']?([\d%]+)["|\']?#i', $images['img_tag'][ $index ], $width_string ) ) {
```

Reply:
> Can't we just match a literal space character?

I tried to leave this as simple as I could, but given that the space is not required (`<img src='https://my-domain.com/wp-content/uploads/2019/11/image.png'width="100px" />` is still valid HTML syntax) we need to match other combinations.